en/developer_guide/external_media_domain.rst

```diff
@@ -0,0 +1,26 @@
+Adding an external media domain
+===============================
+
+Configuration
+~~~~~~~~~~~~~
+
+Sometimes, developers want an external media domain to improve websites performances.
+Open Orchestra allow to add a domain really easily in configuration:
```

Comment:
Open Orchestra gives you an easy way to add an external media domain in the configuration